templates/inventoryitem.json

```diff
@@ -1,12 +1,13 @@
 [
   {
-    "_id": "{{@ system @}}-{{@ datatype @}}-all",
+    "_id": "{{@ system @}}-inventoryItem-all",
```

Comment:
Is there a reason for why the name of the datatype is hardcoded for this type?

Reply:
Fixed